babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
[C] I'd prefer to go with typescript off the bat

Reply:
Thanks will update